CONTRIBUTING.md

```diff
@@ -2,11 +2,13 @@
 
 Contributors to ECMAScript and TC39 are expected to follow our [Code of Conduct](https://tc39.es/code-of-conduct/).
 
+**Please do not open issues or pull requests in this repository to suggest new features.** See the [new feature proposals](#new-feature-proposals) section below for more details.
```

Comment:
Cosmetic issue: bolded body text is evidently the same style as a third-level heading -- can we bump all the headings up by two levels while we're at it?

Reply:
Good thought, done.